docs/docs/reference/Query_Syntax.md

```diff
@@ -3,130 +3,139 @@ title: "Query syntax"
 linkTitle: "Query syntax"
 weight: 1
 description: >
-    Details of the query syntax
+    Learn how to use search query syntax
 ---
 
-# Search Query Syntax
+You can use simple syntax for complex queries using these rules:
 
-We support a simple syntax for complex queries with the following rules:
-
-* Multi-word phrases simply a list of tokens, e.g. `foo bar baz`, and imply intersection (AND) of the terms.
-* Exact phrases are wrapped in quotes, e.g `"hello world"`.
-* OR Unions (i.e `word1 OR word2`), are expressed with a pipe (`|`), e.g. `hello|hallo|shalom|hola`.
-* NOT negation (i.e. `word1 NOT word2`) of expressions or sub-queries. e.g. `hello -world`. As of version 0.19.3, purely negative queries (i.e. `-foo` or `-@title:(foo|bar)`) are supported.
-* Prefix/Infix/Suffix matches (all terms starting/containing/ending with a term) are expressed with a `*`. For performance reasons, a minimum term length is enforced (2 by default, but is configurable).
-* Wildcard pattern matches: `w'foo*bar?'`.
-* A special "wildcard query" that returns all results in the index - `*` (cannot be combined with anything else).
+* Multiword phrases are lists of tokens, for example, `foo bar baz`, and imply intersection (AND) of the terms.
+* Exact phrases are wrapped in quotes, for example, `"hello world"`.
+* `OR` unions are expressed with a pipe (`|`), for example, `hello|hallo|shalom|hola`.
```

Comment:
There are related fixes in DIALECT 2
See "Examples of queries impacted" in [v2.4.3](https://github.com/RediSearch/RediSearch/releases/tag/v2.4.3/) Release Notes

> Consider a simple query with negation -hello world.
> ...

> Another example illustrating the differences in parser behavior
> hello world | "goodbye" moon.
> 
> In DIALECT 1, this query is interpreted as searching for (hello world | "goodbye") moon.
> In DIALECT 2, this query is interpreted as searching for either hello world OR "goodbye"  moon
> 
>